### PR TITLE
MINOR: Remove jcenter and use mavenCentral

### PR DESCRIPTION
**Pull request by bbejeck**
### Description

Found one tutorial that still references `jcenter`.

@ybyzek your call to merge for this release or not.

### CODE REVIEW

**Review by ybyzek — COMMENTED** [review]
Do these instances of `jcenter` need to be replaced as well?

```
_includes/tutorials/window-final-result/kstreams/code/settings.gradle:        jcenter()
_includes/tutorials/window-final-result/kstreams/code/settings.gradle:            name "JCenter Gradle Plugins"
_includes/tutorials/produce-consume-lang/scala/code/settings.gradle:        jcenter()
_includes/tutorials/produce-consume-lang/scala/code/settings.gradle:            name "JCenter Gradle Plugins"
```

**Comment by gAmUssA** [comment]
@ybyzek @bbejeck we should use `gradlePluginPortal()` call instead of bintray or maven central.

**Comment by bbejeck** [comment]
```

**Comment by bbejeck** [comment]
I only checked the `build.gradle` files before
```
_includes/tutorials/window-final-result/kstreams/code/settings.gradle:        jcenter()
_includes/tutorials/window-final-result/kstreams/code/settings.gradle:            name "JCenter Gradle Plugins"
_includes/tutorials/produce-consume-lang/scala/code/settings.gradle:        jcenter()
_includes/tutorials/produce-consume-lang/scala/code/settings.gradle:   
```
These entries are `plugin management` entries in `settings.gradle` we don't use this anywhere else in our tutorials so I'm inclined to remove these.  IMHO it's better to have everything in the build file.

File: _includes/tutorials/produce-consume-lang/scala/code/settings.gradle [hunk 1]
```diff
@@ -1,13 +1,3 @@
-pluginManagement {
-    repositories {
-        gradlePluginPortal()
-        jcenter()
```
Comment (gAmUssA) [inline _includes/tutorials/produce-consume-lang/scala/code/settings.gradle:4]: @bbejeck only `gradlePluginPortal()` should be there. or just remove `jcenter()` and keep rest

**Comment by bbejeck** [comment]
> @ybyzek @bbejeck we should use gradlePluginPortal() call instead of bintray or maven central.

@gAmUssA I removed the entries from `settings.gradle` altogether, to keep things consistent across projects

**Review by ybyzek — APPROVED** [review]
LGTM, This PR makes the KT consistent with other KTs.  I propose we track @gAmUssA 's [comment]([url](https://github.com/confluentinc/kafka-tutorials/pull/805#issuecomment-824122735)) in another issue.

**Comment by bbejeck** [comment]
There is a single test failure, but it's unrelated to the changes in this PR.

**Pull request merged**